.github/workflows/pr-realm-js.yml

```diff
@@ -108,6 +108,11 @@ jobs:
         if: ${{ runner.os == 'Windows' }}
         uses: ilammy/msvc-dev-cmd@v1
 
+      - name: Setup CMake
+        uses: jwlawson/actions-setup-cmake@v2
```

Comment:
For what it's worth [this comment](https://github.com/actions/runner-images/issues/9680#issuecomment-2052018532) suggests `lukka/get-cmake` is faster 🤔